terraform_cft/00_projects/config.tf

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2019 Google LLC. This software is provided as-is, without warranty
+ * or representation for any use or purpose. Your use of it is subject to your 
+ * agreement with Google.  
+ */
+provider "google" {
+  version     = "~> 2.16"
+  credentials = "${file(var.gcp_credentials_path)}"
```

Comment:
Note that I usually encourage using ADC or env variables for more flexibility.

Reply:
Sorry, what is ADC? Do you have an example of this?

Reply:
Application Default Credentials: https://cloud.google.com/docs/authentication/production?_ga=2.70196368.-1476853007.1578333150